rllib/algorithms/appo/tests/tf/test_appo_learner.py

```diff
@@ -67,6 +67,9 @@ def test_appo_loss(self):
                 _enable_rl_module_api=True,
             )
         )
+        # We have to set exploration_config here manually because setting it through
```

Comment:
Add `TODO (Artur): deprecate exploration_config for APPO when RLModuleAPI is used. `